clang/lib/3C/CastPlacement.cpp

```diff
@@ -20,77 +20,80 @@ using namespace clang;
 bool CastPlacementVisitor::VisitCallExpr(CallExpr *CE) {
   // Get the constraint variable for the function.
   Decl *CalleeDecl = CE->getCalleeDecl();
-  FVConstraint *FV = nullptr;
   FunctionDecl *FD = dyn_cast_or_null<FunctionDecl>(CalleeDecl);
-  if (FD) {
-    FV = Info.getFuncConstraint(FD, Context);
-    assert("Function has no definition" && FV != nullptr);
-  } else if (isa_and_nonnull<VarDecl>(CalleeDecl)) {
-    CVarOption Opt = Info.getVariable(cast<VarDecl>(CalleeDecl), Context);
-    if (Opt.hasValue()) {
-      if (isa<PVConstraint>(Opt.getValue()))
-        FV = cast<PVConstraint>(Opt.getValue()).getFV();
-      else
-        FV = dyn_cast<FVConstraint>(&Opt.getValue());
-    }
+
+  // Find a FVConstraint for this call. If there is more than one, then they
+  // will have been unified during constraint generation, so we can use any of
+  // them.
```

Comment:
Per your commit message: You replaced code to find the `FVConstraint` with code used in ConstraintBuilder. Instead of cutting & pasting, is there a way to actually call the same function? Maybe this code should be in `ConstraintResolver` or something like that?